script.js

```diff
@@ -1,29 +1,28 @@
+var plays = 0;
+var have_winner = false;
 
-var jogadas = 0;
-var ha_vencedor = false;
-
-var marca_posicao =  [
+var insert_position =  [
     [false, false, false],
     [false, false, false], 
     [false, false, false], 
 ];
 
-document.getElementById('vez').innerHTML = 'Vez de O';
-document.getElementById('jogadas').innerHTML ='Jogadas feitas: '+jogadas;
+document.getElementById('turn').innerHTML = 'Vez de O';
+document.getElementById('plays').innerHTML ='Jogadas feitas: '+ plays;
 
 function getRandomInt(min, max) {
     min = Math.ceil(min);
     max = Math.floor(max);
     return Math.floor(Math.random() * (max - min)) + min;
 }
-var linhaRandom = getRandomInt(0, 3);
-var colunaRandom = getRandomInt(0, 3);
+var lineRandom = getRandomInt(0, 3);
```

Comment:
_Line_ is a noun (_substantivo_ in Portuguese) and _random_ is an adjective.

In Portuguese, we use a _noun + adjective_, but in English, we have the opposite: _adjective + noun_.

So, this way, we should have `randomLine` rather than `lineRandom`.